framework/grid/ActionColumn.php

```diff
@@ -142,9 +142,9 @@ public function init()
      */
     protected function initDefaultButtons()
     {
-        $this->initDefaultButton('view', 'eye-open');
-        $this->initDefaultButton('update', 'pencil');
-        $this->initDefaultButton('delete', 'trash', [
+        $this->initDefaultButton('view', '👁');
+        $this->initDefaultButton('update', '✏');
+        $this->initDefaultButton('delete', '🗑', [
```

Comment:
I'd use ❌, ✎, 👁

Reply:
https://github.com/yiisoft/yii2/issues/16359#issuecomment-403242885

Reply:
I would leave 🗑 - it is more similar to previous icon and IMO fits better for delete action.

Reply:
OK.